services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/oauth/OpenIDConnectManagerImpl.java

```diff
@@ -375,7 +373,9 @@ public UserAuthorization getUserAuthorization(String oauthToken) {
 			throw new IllegalArgumentException("Missing 'audience' value in the OAuth Access Token.");
 		}
 
-		validateClientVerificationStatus(oauthClientId);
+		if (!oauthClientId.equals(AuthorizationConstants.SYNAPSE_OAUTH_CLIENT_ID)) {
```

Comment:
Put this into the validateCLientVerificationStatus

Reply:
Done.